pyproject.toml

```diff
@@ -40,14 +40,13 @@ disable = [
     "missing-module-docstring",
     "consider-using-f-string",
     "too-many-branches",
+    "too-many-arguments",
     "broad-exception-caught",
     "no-else-break",
     "no-else-return",
     "invalid-name",
     "import-error",
     "unnecessary-pass",
-    "unspecified-encoding",
-    "protected-access",
```

Comment:
May I know why this line "protected-access" is removed in the last rebase? Which is introduced in the last merged PR: https://github.com/xapi-project/xen-api/pull/5490/files
Seems it is needed to pass CI for nbd_client_manager.py, but it is not required now?

Reply:
OK, I can remove it.